saleor/dashboard/menu/views.py

```diff
@@ -120,7 +119,6 @@ def menu_item_create(request, menu_pk, root_pk=None):
         msg = pgettext_lazy(
             'Dashboard message', 'Added menu item %s') % (menu_item,)
         messages.success(request, msg)
-        update_menu(menu)
```

Comment:
Why is that? The menu is kept as `JSON` and we need to update it each time it changed.

Reply:
Hm... Very strange. It seems that I accidentally restored some changes from disk which caused these lines to be deleted and I missed it when reviewed my own commit. I apologize for inconvenience and will correct it tomorrow